test/unrunnableTestCases.csv

```diff
@@ -1352,115 +1300,43 @@ op-concatnni2args-2,Error: FOCA0003: can not cast 303884545991464527 to xs:integ
-op-duration-equal-25,AssertionError: Expected XPath (xs:duration("P36D") ne xs:duration("P36D")) or fn:false() to resolve to false: expected true to be false
-op-duration-equal-26,Error: XPTY0004: Values to compare are not of the same type
-op-duration-equal-27,Error: XPTY0004: Values to compare are not of the same type
-op-yearMonthDuration-equal2args-1,AssertionError: Expected XPath xs:yearMonthDuration("P0Y0M") eq xs:yearMonthDuration("P0Y0M") to resolve to true: expected false to be true
-op-yearMonthDuration-equal2args-6,AssertionError: Expected XPath xs:yearMonthDuration("P0Y0M") ne xs:yearMonthDuration("P0Y0M") to resolve to false: expected true to be false
-op-yearMonthDuration-equal2args-12,AssertionError: Expected XPath xs:yearMonthDuration("P1000Y6M") le xs:yearMonthDuration("P0Y0M") to resolve to false: expected true to be false
-op-yearMonthDuration-equal2args-13,AssertionError: Expected XPath xs:yearMonthDuration("P2030Y12M") le xs:yearMonthDuration("P0Y0M") to resolve to false: expected true to be false
-op-yearMonthDuration-equal2args-19,AssertionError: Expected XPath xs:yearMonthDuration("P0Y0M") ge xs:yearMonthDuration("P1000Y6M") to resolve to false: expected true to be false
-op-yearMonthDuration-equal2args-20,AssertionError: Expected XPath xs:yearMonthDuration("P0Y0M") ge xs:yearMonthDuration("P2030Y12M") to resolve to false: expected true to be false
-op-dayTimeDuration-equal2args-1,AssertionError: Expected XPath xs:dayTimeDuration("P0DT0H0M0S") eq xs:dayTimeDuration("P0DT0H0M0S") to resolve to true: expected false to be true
-op-dayTimeDuration-equal2args-6,AssertionError: Expected XPath xs:dayTimeDuration("P0DT0H0M0S") ne xs:dayTimeDuration("P0DT0H0M0S") to resolve to false: expected true to be false
-op-dayTimeDuration-equal2args-12,AssertionError: Expected XPath xs:dayTimeDuration("P15DT11H59M59S") le xs:dayTimeDuration("P0DT0H0M0S") to resolve to false: expected true to be false
-op-dayTimeDuration-equal2args-13,AssertionError: Expected XPath xs:dayTimeDuration("P31DT23H59M59S") le xs:dayTimeDuration("P0DT0H0M0S") to resolve to false: expected true to be false
-op-dayTimeDuration-equal2args-19,AssertionError: Expected XPath xs:dayTimeDuration("P0DT0H0M0S") ge xs:dayTimeDuration("P15DT11H59M59S") to resolve to false: expected true to be false
-op-dayTimeDuration-equal2args-20,AssertionError: Expected XPath xs:dayTimeDuration("P0DT0H0M0S") ge xs:dayTimeDuration("P31DT23H59M59S") to resolve to false: expected true to be false
+op-divide-dayTimeDuration2args-2,AssertionError: xs:dayTimeDuration("P15DT11H59M59S") div xs:double("-1.7976931348623157E308"): expected '-PT7.449541715597465e-303S' to equal 'PT0S'
+op-divide-dayTimeDuration2args-3,AssertionError: xs:dayTimeDuration("P31DT23H59M59S") div xs:double("-1.7976931348623157E308"): expected '-PT1.5379704947317132e-302S' to equal 'PT0S'
```

Comment:
Nope, issue with precision. Every integer above Number.MAX_SAFE_INTEGER and below Number.MIN_SAFE_INTEGER will throw FOTD0002, however, a check for (-)0.12345e-300 should be implemented.